glide.yaml

```diff
@@ -10,7 +10,7 @@ import:
 - package: github.com/opentracing/opentracing-go
   version: ^1
 - package: github.com/uber/tchannel-go
-  version: ^1.3
+  version: dev
```

Comment:
Pinned temporarily. The change is in a TEMP commit which I intend to replace when we release TChannel (tomorrow)